README.md

```diff
@@ -1 +1,20 @@
-# Cproject
\ No newline at end of file
+# C project ( C for College)
+
+This is a feedback system project. Through this project, we will improve the outcome of the feedback's given by the
```

Comment:
Typo "feedback's" instead of "feedbacks" :)

Reply:
I did it like yours 
 but on grammerly it is giving as invalid

Reply:
Because using "the" is invalid with plural "feedbacks" :) Change "the feedback's" to "feedbacks".